src/Uno.UI.Runtime.Skia.Gtk/GTK/GtkCoreWindowExtension.cs

```diff
@@ -41,7 +41,7 @@ public void ReleasePointerCapture()
 		public void SetPointerCapture()
 			=> this.Log().Error("Pointer capture is not supported on GTK");
 
-		public GtkUIElementPointersSupport(object owner)
+		public GtkCoreWindowExtension(object owner)
```

Comment:
Is this a binary breaking change?

Reply:
There's no good reason for those types to be public, I'll change the others.